packages/contracts/test/diamond/facets/UbiquityPoolFacet.t.sol

```diff
@@ -329,6 +329,27 @@ contract UbiquityPoolFacetTest is DiamondTestSetup {
         );
     }
 
+    function testMintDollar_ShouldRevert_IfZeroCollateralAvailable() public {
```

Comment:
👀 ,

Reply:
@molecula451 this is related to https://github.com/ubiquity/ubiquity-dollar/pull/892#discussion_r1476177155